evalml/utils/cli_utils.py

```diff
@@ -118,18 +119,24 @@ def standardize_format(packages, ignore_packages=None):
     for package in packages:
         if package.name in ignore_packages:
             continue
-        name = CONDA_TO_PIP_NAME.get(package.name, package.name)
-        if package.specs:
-            all_specs = ",".join(["".join(spec) for spec in package.specs])
+        name = package.name
+        if convert_to_conda and name in CONDA_TO_PIP_NAME:
+            name = CONDA_TO_PIP_NAME.get(package.name)
```

Comment:
Maybe not a big deal, but should we add a test (or parameterize the existing test if we can) to hit this line?